source/fundamentals/linq.txt

```diff
@@ -249,6 +249,33 @@ The result of the preceding example contains the following documents:
     { "name" : "Crystal Room", "cuisine" : "Italian" }
     { "name" : "Forlinis Restaurant", "cuisine" : "Italian" }
 
+$sample
+~~~~~~~
+
+The ``$sample`` aggregation stage returns a random sample of documents from a 
+collection. The following example shows how to generate a ``$sample`` stage by using
+LINQ: 
+
+.. code-block:: csharp
+   :emphasize-lines: 4
+
+   var query = queryableCollection
+       .Aggregate()
+       .Sample(4)
+       .ToList();
+
+The result of the preceding example contains the following documents:
+
+.. code-block:: json
+
+   // Results Truncated
+
+   { "title" : "State of Play", "languages" : ["English"]}
+   { "title" : "Bhopal: A Prayer for Rain", "languages" : ["English", "Hindi"] }
+   { "title" : "Dead Presidents", "languages" : ["English"]}
+   { "title" : "The Rains Came", "languages" : ["English"] }
```

Comment:
The results from the code samples in this page seem to be using the `restaurants` collection from the `sample_restaurants` database. However, the results here seem to be from the `movies` collection in the `sample_mflix` database. You should update the results here.